Makefile

```diff
@@ -147,6 +147,15 @@ test-python-integration-aim: build ## run the Aim python integration tests.
 	@echo ">>> Running Aim python integration tests."
 	tests/integration/python/aim/test.sh
 
+PHONY: test-k6-performance
+test-k6-performance: service-restart
+	@echo ">>> Running k6 performance tests."
+	@COMPOSE_FILE=$(COMPOSE_FILE) COMPOSE_PROJECT_NAME=$(COMPOSE_PROJECT_NAME) \
+	    docker-compose run performance-tests
+	@COMPOSE_FILE=$(COMPOSE_FILE) COMPOSE_PROJECT_NAME=$(COMPOSE_PROJECT_NAME) \
```

Comment:
i think the "down" step isn't needed here (previously only in the `service-clean` target)

Reply:
Oh okay, I set it there to remove the containers once the process is complete, but that's fine, I will remove it